src/components/uiTypeAhead.vue

```diff
@@ -172,7 +210,11 @@ export default {
         &:hover{
             background-color:  $ms-color-themeLight; //To overwrite unselected hover
         }
-    }
-   
+    } 
+
+    .selected {
+        color: $ms-color-black;
+        background-color: rgb(123, 152, 207);
```

Comment:
Is it possible to move this to a variable? Are we following `RGB` or `Hex Color` in this repo?

Reply:
I have since removed this rule as it was only there for troubleshooting. Both the defaults and saved data are mostly in `hex` format.